spec/features/admin_creates_trip_spec.rb

```diff
@@ -18,21 +18,21 @@
         click_link "Create a Trip"
 
         expect(current_path).to eq(new_admin_trip_path)
-        within("h2") { expect(page).to have_content("Create a New Trip") }
+        expect(page).to have_content("Create a New Trip")
       end
     end
 
     context "create-trip page" do
       it "creates a trip with valid params" do
         Destination.create(continent: "Asia", description: "description",
-                    lat: 31.8369, long: -77.6847, image_url: "asia.jpg" )
+                     image_url: "asia.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.